src/sentry/static/sentry/app/views/settings/project/projectProcessingIssues.jsx

```diff
@@ -453,6 +435,10 @@ class ProjectProcessingIssues extends React.Component {
   }
 }
 
+const StyledPanelAlert = styled(PanelAlert)`
+  grid-column: 1/5;
```

Comment:
A nice solution I found for spanning across a grid is to use `grid-column: 1/-1;`

That way you don't have to update this for the exact column count.